lib/tasks/data_migrations/convert_terms_and_conditions_link_setting.rake

```diff
@@ -2,16 +2,17 @@ namespace :data_migrations do
   task convert_terms_and_conditions_link_setting: :environment do
```

Comment:
Suppose maybe we should get rid of all the old data migration, how do you think?

Reply:
That's what I was thinking also. They don't seem to be relevant anymore and I doubt someone has few months old registry running somewhere in the closet